.dict_custom.txt

```diff
@@ -119,4 +119,6 @@ indexable
 traceback
 STC
 gFTL
-CMake
\ No newline at end of file
+CMake
+gcd
+lcm
```

Comment:
Code should be quoted. Acronyms should be capitalised. I think that here you should be using one of:
```
`gcd`/`lcd`
greatest/lowest common denominator
```

Reply:
Done [4dbf41c](https://github.com/pyccel/pyccel/pull/1986/commits/4dbf41c7fb7a2744fe8a10314cee2d5fcced65a7)